v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -22,24 +22,28 @@
 
 /** The {@link ElasticsearchWriteOptions} with the common write options for Elasticsearch. * */
 public interface ElasticsearchWriteOptions extends PipelineOptions {
-  @Description(
-      "Comma separated list of Elasticsearch nodes to connect to, ex: http://my-node1,http://my-node2")
+  @Description("Elasticsearch URL in format http://hostname:[port] or Base64 encoded CloudId")
   @Validation.Required
-  String getTargetNodeAddresses();
+  String getConnectionUrl();
 
-  void setTargetNodeAddresses(String targetNodeAddresses);
+  void setConnectionUrl(String connectionUrl);
 
-  @Description("The index toward which the requests will be issued, ex: my-index")
+  @Description("Username for Elasticsearch endpoint")
   @Validation.Required
-  String getWriteIndex();
+  String getWriteElasticsearchUsername();
```

Comment:
Can you drop the word write in the option? `getWriteElasticsearchUsername` to `getElasticsearchUsername`